libvast/vast/io/save.hpp

```diff
@@ -0,0 +1,28 @@
+/******************************************************************************
+ *                    _   _____   __________                                  *
+ *                   | | / / _ | / __/_  __/     Visibility                   *
+ *                   | |/ / __ |_\ \  / /          Across                     *
+ *                   |___/_/ |_/___/ /_/       Space and Time                 *
+ *                                                                            *
+ * This file is part of VAST. It is subject to the license terms in the       *
+ * LICENSE file found in the top-level directory of this distribution and at  *
+ * http://vast.io/license. No part of VAST, including this file, may be       *
+ * copied, modified, propagated, or distributed except according to the terms *
+ * contained in the LICENSE file.                                             *
+ ******************************************************************************/
+
+#include "vast/byte.hpp"
+#include "vast/fwd.hpp"
+#include "vast/span.hpp"
+
+#include "caf/fwd.hpp"
+
+namespace vast::io {
+
+/// Safely writes a buffer to a file on disk.
```

Comment:
The reader doesn't know how to interpret "safely" here. Please provide an adequate description of what happens.